onnx/defs/schema.cc

```diff
@@ -699,21 +699,14 @@ void OpSchema::ParseAndSetTypes(
 OpSchema& OpSchema::SetContextDependentFunctionBodyBuilder(
     ContextDependentFunctionBodyBuilder functionBuilder,
     int opset_version) {
-  if (opset_version == OpSchema::kUninitializedSinceVersion && since_version_ != OpSchema::kUninitializedSinceVersion) {
```

Comment:
nit: I wonder if we should have an assert to ensure that opset_verion is > 0 (that is, not equal to `OpSchema::kUninitializedSinceVersion`)?  Just to catch it if user passes that value?

Reply:
Good question - we had a similar discussion in https://github.com/onnx/onnx/pull/5906#discussion_r1479154274 as well.